container-definitions/app.json.tpl

```diff
@@ -30,10 +33,10 @@
     ],
     %{ endif }
     %{ if environment != "[]" }
-    "environment": "${environment},
```

Comment:
im slighty confused by this change.

Reply:
That was a fix from the previous PR ... It shouldn't have the `"`

Reply:
(`environment` hasn't been used until this PR)